go/src/koding/klientctl/metrics/bolt.go

```diff
@@ -0,0 +1,110 @@
+package metrics
+
+import (
+	"io"
+	"strconv"
+	"time"
+
+	"github.com/boltdb/bolt"
+)
+
+const bucketName = "metrics"
+
+var _ io.WriteCloser = &BoltQueue{}
+
+// BoltQueue writes and reads to BoltDB
+type BoltQueue struct {
+	db         *bolt.DB
+	bucketName []byte
```

Comment:
We could make global bucketName a []byte, remove this field and use global var instead.